lib/submit-measurement.js

```diff
@@ -0,0 +1,19 @@
+/**
+ *
+ * @param {{retrievalSucceeded: boolean}} measurement
+ * @param {typeof globalThis.fetch} fetch
+ */
+export const submit = async (measurement, fetch = globalThis.fetch) => {
+  const res = await fetch('https://api.checker.network/walrus/measurement', {
+    method: 'POST',
+    headers: {
+      'Content-Type': 'application/json'
+    },
+    body: JSON.stringify(measurement)
+  })
+  if (!res.ok) {
```

Comment:
could use assertOkResponse too

Reply:
Resolved in https://github.com/CheckerNetwork/walrus-checker/pull/13/commits/a76b3a5e63e3f77b956f36aef0a6e44ebfb64a2f